tests/acceptance/clinical_browser.js

```diff
@@ -180,15 +180,15 @@ function clinVarSearchByGene(driver){
 }
 function clinVarSearchByTrait(driver){
     driver.findElement(By.xpath("//div[contains(@id,'ClinVarPositionFilterFormPanel')]//div[contains(@id,'selectFilter-trigger-picker')]")).click();
-    driver.findElement(By.xpath("//li[text()='Chromosomal Location']")).click();
-    driver.findElement(By.name("clinvarRegion")).clear();
-    driver.findElement(By.name("clinvarRegion")).sendKeys("2:47000000-48000000");
+    // driver.findElement(By.xpath("//li[text()='Chromosomal Location']")).click();
+    // driver.findElement(By.name("clinvarRegion")).clear();
+    // driver.findElement(By.name("clinvarRegion")).sendKeys("2:47000000-48000000");
```

Comment:
Changes made. selectAnnotCT is coming from eva.js  when the component is initially rendered. this is for loading pre selected values like when page refreshed or relaoded